README.md

```diff
@@ -107,6 +107,16 @@ After running clusterctl, a minikube instance is going to be running locally, wh
 - To tail cluster actuator logs, `kubectl get po -o name | grep clusterapi-controllers | xargs kubectl logs -c aws-machine-controller -f`.
 - To tail machine actuator logs, `kubectl get po -o name | grep clusterapi-controllers | xargs kubectl logs -c aws-cluster-controller -f`.
 
+#### CloudFormation
```

Comment:
Should we rephrase this to be something like `Setting up prerequisites with CloudFormation`? What do you think about adding a list of resources (with bullet points) that are needed?

Reply:
#161 replaces this README.md, so I might as well delete it here